sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/CreateNotification.groovy

```diff
@@ -47,12 +47,13 @@ class CreateNotification implements CreateNotificationInput {
             projectsWithSamples.each { project ->
                 project.title = projectsWithTitles.get(project.code)
             }
-
             //add notifications with create notification method (directly adds notifications to list)
             projectsWithSamples.each { project ->
                 addNotificationForProject(project)
             }
-            output.createdNotifications(notifications)
+            if (!notifications.isEmpty()) {
+                output.createdNotifications(notifications)
+            }
```

Comment:
I think it would make sense to always call the output and let the "app" layer decide how to handle that. But I am happy to discuss this because I am really unsure. It just feels strange that the output interface is not called

Reply:
I also was unsure how to handle this here but in theory we don't present anything to the output interface if no email was generated